cms/djangoapps/contentstore/management/commands/tests/test_delete_course.py

```diff
@@ -60,3 +63,32 @@ def test_course_deleted(self):
             patched_yes_no.return_value = True
             call_command('delete_course', 'TestX/TS01/2015_Q1')
             self.assertIsNone(modulestore().get_course(SlashSeparatedCourseKey("TestX", "TS01", "2015_Q1")))
+
+    def test_course_deletion_with_keep_instructors(self):
+        """
+        Test keep-instructors option,use of this option delete course from mongo store without
+        removing instructors.
+        """
+        course_id = SlashSeparatedCourseKey("TestX", "TS01", "2015_Q1")
+        staff_user = User.objects.create(
+            username='test_staff',
```

Comment:
I think it's good to Check for `staff_user` either its create or not before using it.